plugins/airbrake/src/plugin.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createPlugin,
+  createRoutableExtension,
+} from '@backstage/core-plugin-api';
+
+import { rootRouteRef } from './routes';
+
+export const airbrakePlugin = createPlugin({
+  id: 'airbrake',
+  routes: {
+    root: rootRouteRef,
+  },
+});
+
+export const AirbrakeWidget = airbrakePlugin.provide(
+  createRoutableExtension({
```

Comment:
Is this intended to go on entity pages? In that case it should probably be `EntityAirbrakeCard` or `EntityAirbrakeContent` instead. If it's supposed to be a full-page thing then it should contain the full `<Page>` layout and be called `AirbrakePage` or something like that. And if it's just some content that you can put on any page then it would just be a plain component extension rather than a routable one.

Reply:
It should be like Sentry, where it shows up as a tab on a component page if the annotation exists on catalog-info.yaml.

Should I just remove everything to do with this routable extension? And just keep the `createPlugin()` call?

Apologies, I don't fully get the generated code on here, so any help would be appreciated!

Reply:
Ah alright, if it's a tab on the component page then `EntityAirbrakeContent` would be the desired name, and keep the rest as is 👍

Reply:
Done